include/fpm/ios.hpp

```diff
@@ -737,4 +737,44 @@ std::basic_istream<CharT, Traits>& operator>>(std::basic_istream<CharT, Traits>&
 
 }
 
+#if __cplusplus >= 202002L
+#   include <version>
+#endif
+#ifdef __cpp_lib_format
+#   include <format>
+#   include <sstream>
+template<typename CharT, typename B, typename I, unsigned int F, bool R>
+struct std::formatter<fpm::fixed<B, I, F, R>, CharT>
+{
+    bool showpos = false;
+
+    template<class ParseContext>
+    constexpr typename ParseContext::iterator parse(ParseContext& ctx)
+    {
+        auto it = ctx.begin();
+        if(it != ctx.end() && *it == '+')
+        {
+            showpos = true;
```

Comment:
I think if this now exposes `std::format` capabilities, it should be done completely and fully. Would you be able to add support for the full float formatting specification? https://en.cppreference.com/w/cpp/utility/format/spec

Reply:
I will do my best but I've never wrote this complex formatting. I just hope I'll be able to put the output together with existing `operator<<` (that is a complex one).